helmfile.d/grafana.yaml

```diff
@@ -9,3 +9,5 @@ releases:
         - "../config/default/grafana.yaml"
       secrets:
         - "../secrets/config/grafana/secrets.yaml"
+bases:
+  - 00-repositories.yaml
```

Comment:
What's that file?

Reply:
the repos, apparently the default parent implied way is deprecated, without it, it wouldn't generate the lock file, (i manually edited the file so downgrade, hopefully it works)